src/screens/OrganizationEvents/OrganizationEvents.tsx

```
@@ -196,41 +205,35 @@ function organizationEvents(): JSX.Element {
     ) {
       try {
+        const startTimeParts = formState.startTime.split(':');
+        const endTimeParts = formState.endTime.split(':');
+
+        const input = {
+          name: formState.title,
+          description: formState.eventdescrip,
+          startAt: alldaychecked
+            ? dayjs(startDate).startOf('day').toISOString()
+            : dayjs(startDate)
+                .hour(parseInt(startTimeParts[0]))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)